grape-swagger.gemspec

```diff
@@ -15,8 +15,8 @@ Gem::Specification.new do |s|
   s.metadata['rubygems_mfa_required'] = 'true'
 
   s.required_ruby_version = '>= 3.0'
-  s.add_runtime_dependency 'grape', '>= 1.7', '< 3.0'
-  s.add_runtime_dependency 'rack-test', '~> 2'
+  s.add_dependency 'grape', '>= 2.2.0', '< 3.0'
+  s.add_dependency 'rack-test', '~> 2'
```

Comment:
The switch from `add_runtime_dependency` to `add_dependency` was made to satisfy Rubocop.